src/Compilers/CSharp/Portable/Compilation/CSharpCompilation.cs

```diff
@@ -2403,27 +2404,27 @@ internal void AddInterception(string filePath, int line, int character, Location
                 factoryArgument: (AttributeLocation: attributeLocation, Interceptor: interceptor));
         }
 
+        internal bool InterceptorsDiscoveryComplete;
```

Comment:
>internal bool InterceptorsDiscoveryComplete; [](http://example.com/codeflow?start=8&length=44)

Consider to not "sprinkle" field declarations all over the file. I think we prefer placing all field declarations at the top of a type declaration, but, since this is already "broken" in this type, consider at least placing this declaration next to declaration of ```_interceptions```. At least related fields will be next to each other.